.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,80 @@
+name: Publish package to Release Artifactory
+on:
+  release:
+    types: [created]
+env:
+  VAULT_ADDR: https://clotho.broadinstitute.org:8200
+  ARTIFACTORY_ACCOUNT_PATH: secret/dsp/accts/artifactory/dsdejenkins
+jobs:
+  publish_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: develop
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Pull Vault image
+        run: docker pull vault:1.1.0
+      # Currently, there's no way to add capabilities to Docker actions on Git, and Vault needs IPC_LOCK to run.
+      - name: Get Vault token
+        id: vault-token-step
+        run: |
+          VAULT_TOKEN=$(docker run --rm --cap-add IPC_LOCK \
+            -e "VAULT_ADDR=${VAULT_ADDR}" \
+            vault:1.1.0 \
+            vault write -field token \
+              auth/approle/login role_id=${{ secrets.VAULT_APPROLE_ROLE_ID }} \
+              secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
+          echo ::set-output name=vault-token::$VAULT_TOKEN
+          echo ::add-mask::$VAULT_TOKEN
+      - name: Get artifactory credentials from Vault
+        id: vault-secret-step
+        run: |
+          ARTIFACTORY_USERNAME=$(docker run --rm --cap-add IPC_LOCK \
+            -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
+            -e "VAULT_ADDR=${VAULT_ADDR}" \
+            vault:1.1.0 \
+            vault read -field username ${ARTIFACTORY_ACCOUNT_PATH})
+          ARTIFACTORY_PASSWORD=$(docker run --rm --cap-add IPC_LOCK \
+            -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
+            -e "VAULT_ADDR=${VAULT_ADDR}" \
+            vault:1.1.0 \
+            vault read -field password ${ARTIFACTORY_ACCOUNT_PATH})
+          echo ::set-output name=artifactory-username::$ARTIFACTORY_USERNAME
+          echo ::add-mask::$ARTIFACTORY_USERNAME
+          echo ::set-output name=artifactory-password::$ARTIFACTORY_PASSWORD
+          echo ::add-mask::$ARTIFACTORY_PASSWORD
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+#      TODO(yonghao): add this back once we have test
+#     - name: Run unit tests
+#        id: unit-test
+#        run: ./gradlew unitTest --scan
+#      - name: Run integration tests
+#        id: integration-test
+#        run: ./gradlew integrationTest --scan
+#      - name: Upload Test Reports
+#        if: always()
+#        uses: actions/upload-artifact@v1
+#        with:
+#          name: Test Reports
+#          path: build/reports/tests
+      - name: "Bump the tag to a new version"
+        uses: databiosphere/github-actions/actions/bumper@v0.0.3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DEFAULT_BUMP: patch
+          RELEASE_BRANCHES: master,develop
```

Comment:
How does this relate to the develop branch specified in the Checkout step above?

Reply:
That is for which branches will generate release tags:
https://github.com/DataBiosphere/github-actions/tree/master/actions/bumper#options

I don't think I will need master here anymore. Leave it here just in case other libraries copy-paste from here but forget to change this line...

Reply:
Okay, up to you. My slight preference is to just put a note in the README to change the branch to master if that's the "main" branch for the repository.